### ISSUE TITLE
No response when running script containing bump2version

### ISSUE THREAD

**Issue by Andy-AO** [PowerShell/vscode-powershell#3576]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md) guide.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.

### Summary

Using `Run: Start Without Debugging` to run a PowerShell script containing the `bump2version` native command may be unresponsive, but running it in another way is perfectly fine.

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Visual Studio Code Version

```console
1.60.1
83bd43bc519d15e50c4272c6cf5c1479df196a4d
x64
```

### Extension Version

```console
ms-vscode.powershell@2021.9.0
```

### Steps to Reproduce

```PowerShell
# main.ps1
bump2version

```

This bug involves the native command `bump2version`, if you don't have it installed, you can run the command `pip install --upgrade bump2version` to install it.

Running `main.ps1` with `Run: Start Without Debugging` in VSCode will cause a longer unresponsiveness, but if you use the path of `main.ps1` entered directly, this problem does not occur.

### Visuals

<img width="523" alt="image_824" src="https://user-images.githubusercontent.com/38101170/134315607-837e432d-5e40-4fbf-9a37-a68d2aa6bc83.png">

**Comment by andschwa** [comment]
I wonder if this is related to similar bugs with some native tools like we see in #2559. My guess is some input handle is hung-up but it's just a guess.

**Comment by Andy-AO** [comment]
> I wonder if this is related to similar bugs with some native tools like we see in #2559. My guess is some input handle is hung-up but it's just a guess.

It may be similar.

**Comment by Andy-AO** [comment]
> I wonder if this is related to similar bugs with some native tools like we see in #2559. My guess is some input handle is hung-up but it's just a guess.

This issue has been around for about a year and there is no sign of a solution, it feels like it's affecting the experience of using it, is it a low priority?

**Comment by andschwa** [comment]
It's not a low priority, it's one of many priorities and it's one of the hardest to solve.

**Comment by Andy-AO** [comment]
If it is simply not responding, then the problem is not too serious. More importantly, even if the operation has been stopped using `^c`, the corresponding folder will be occupied, and unless the whole program is restarted, it will be difficult to lift it, and the problem will be serious if you accidentally press the wrong button.

Moreover, if this problem occurs in debug mode, it is impossible to run the script in vscode using the normal way, and it will all go into debug state.

**Comment by andschwa** [comment]
It's very interesting to me that the programs we're seeing problems with are all interpreted scripts. Both `hg.exe` and `bump2version.exe` are Python scripts (executables that bundle the Python interpreter and the associated script), and `exiftool.exe` is a Perl script bundled in the same manner. That is to say, they are not simply native executables. There seems to be a connection here but I haven't quite figured it out.

**Comment by andschwa** [comment]
@Andy-AO Hi there, we just released [v2022.4.0-preview](https://github.com/PowerShell/vscode-powershell/releases/tag/v2022.4.0-preview)! Could you please try [PowerShell Preview for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell-Preview) and verify this is fixed?

**Comment by andschwa** [comment]
I'm going to assume this was fixed as the root cause was (finally) addressed. Please re-open if not the case!
